radiospectra/net/sources/tests/test_stereo_client.py

```diff
@@ -19,6 +19,7 @@ def client():
     return SWAVESClient()
 
 
+@pytest.mark.skip("Upstream data server removed")
```

Comment:
Will it come back?

Reply:
Nope the job of processing/hosting has moved to a different group/institute and the data is now only available on the [CDPP](https://cdpp-archive.cnes.fr) and I'm not sure how to search that via an API also I think the files are in CDF format.

Reply:
So we should remove these tests and the client for it?

Reply:
Yea I need to check where the data can be obtained and in what formats so want to leave the code until I figure that out, only skipped the tests for the CI really.